main.py

```diff
@@ -138,7 +139,7 @@ async def get_task_status(task_id: str) -> TaskStatus:
 
 
 @app.get("/exceptions/{task_id}")
-async def get_task_exceptions(task_id: str) -> list[str]:
+async def get_task_exceptions(task_id: str) -> dict:
```

Comment:
We may want to add return types to the other tasks as well, I noticed most of them don't have them.